### PR TITLE
#1809: Update to center page content on pages with no sidebar

### PR DESCRIPTION
**Pull request by tzhou3**
I made the changes as specified by issue #1809, please let me know if there is anything I did wrong or that I can help with. Thanks for your time.

### CODE REVIEW

File: app/views/wiki/show.html.erb [hunk 1]
```diff
@@ -25,19 +25,24 @@
...
+<% else %>  
   <% if (@wikis.nil? && @notes.nil?) || @node.has_tag('sidebar:featured') %>
     <%= render :partial => "sidebar/featured" %>
   <% else %>
     <%= render :partial => "sidebar/related" %>
   <% end %>
-<% end %>
+
 
 <div class="col-md-9">
```
Comment (jywarren) [inline app/views/wiki/show.html.erb]: Hi, if you look closely at the original issue, the `<% end %>` should go on this line, not below the `partial wiki/header` line. But you're almost there -- great work so far! 

If you can make these adjustments, I look forward to merging this in!

Comment (tzhou3) [reply app/views/wiki/show.html.erb]: Oops that's embarrassing, I added in the end tag and it should be good now. Sorry for that and thanks again.

**Comment by tzhou3** [comment]
Okay I have implemented the changes that you suggested. I really appreciate your help on this issue. Thanks a lot.

**Comment by jywarren** [comment]
Perfect!!! Great work, a pleasure working with you! 

If you're looking for a new challenge, please take a look at our `help-wanted` list: https://github.com/publiclab/plots2/labels/help-wanted

Thanks!

**Pull request merged**